tests/acceptance/features/webUIWebdavLocks/locks.feature

```diff
@@ -30,19 +30,19 @@ Feature: Locks
 
   @issue-5417
   Scenario: setting a lock shows the display name of a user in the locking details
-    Given user "brand-new-user" has locked folder "simple-folder" setting following properties
+    Given user "brand-new-user" has locked folder "simple-folder" setting the following properties in the server
       | lockscope | shared |
-    And user "brand-new-user" has locked file "data.zip" setting following properties
+    And user "brand-new-user" has locked file "data.zip" setting the following properties in the server
       | lockscope | exclusive |
     When the user re-logs in as "brand-new-user" using the webUI
     Then folder "simple-folder" should be marked as locked by user "New User" in the locks tab of the details panel on the webUI
     And file "data.zip" should be marked as locked by user "New User" in the locks tab of the details panel on the webUI
 
   @issue-5417
   Scenario: setting a lock shows the current changed display name of a user in the locking details
-    Given user "brand-new-user" has locked folder "simple-folder" setting following properties
+    Given user "brand-new-user" has locked folder "simple-folder" setting the following properties in the server
       | lockscope | shared |
-    And user "brand-new-user" has locked file "data.zip" setting following properties
+    And user "brand-new-user" has locked file "data.zip" setting the following the properties in the server
```

Comment:
```suggestion
    And user "brand-new-user" has locked file "data.zip" setting the following properties in the server
```

How did this pass?

Reply:
This scenario is in `expected-to-tail` that must be the reason why it passed. Fixed it in the recent push.